kubernetes/src/main/java/io/kubernetes/client/custom/Quantity.java

```diff
@@ -37,8 +37,9 @@ public Quantity(final BigDecimal number, final Format format) {
     }
 
     public Quantity(final String value) {
```

Comment:
Aside from this PR, it seems weird to me that this constructor calls a static factory method, gets an instance of Quantity, and reassigns the attributes to itself.